agent/stats/queue.go

```diff
@@ -103,6 +103,13 @@ func (queue *Queue) add(rawStat *ContainerStats) {
 			stat.CPUUsagePerc = 100 * cpuUsageSinceLastStat / timeSinceLastStat
 		}
 
+		//calculate per second Network metrics
+		timeSinceLastStatInSeconds := uint64(rawStat.timestamp.Sub(lastStat.Timestamp).Seconds())
```

Comment:
this is going to be a problem for streaming stats, which sometimes might return stats that are less than 1 second apart from each other. I think it would be better to determine the bytes per millisecond and then divide by 1000 to get the per-second average.

Reply:
this would also mean you dont have to change the timestamps of the unit tests (which i think should stay as they are, since we need to support metrics arriving at less than 1-second intervals).

Reply:
> this is going to be a problem for streaming stats

For AWSVPC mode network stats we are only polling stats.

Reply:
Yes it makes sense to reuse `timeSinceLastStat` on line 96.

Reply:
Fixed it. Changed the unit of the per sec metrics to float32 which will help us to calculate the metric for even a small time duration